package-lock.json

```diff
@@ -0,0 +1,19988 @@
+{
```

Comment:
Let's not commit this. I'm using bun for this project.

Reply:
Apologies, totally missed that. Should be good now.

Reply:
No worries!